Moderation_Matrix.md

```diff
@@ -0,0 +1,52 @@
+# Moderation Matrix
+
+## 1 Zap (WARNING) 
+Minor violations such as:
+- "Mini-modding"
+  - Please leave moderation action to the moderators, use ModMail to report rule breaking or if you see something you’re unsure about in the community
+- Asking about topics or projects unrelated to The Odin Project (Rules 11 and 12)
+-Mild unprofessionalism: see this guide https://www.bitrix24.com/about/blogs/communications/instant-messaging-and-group-chat-etiquette-in-a-professional-environme.php 
+  - Drug related conversation (Rule 7)
+  - Special attention to treating everyone with respect
+  - Mental health issues discussion (Rule 12)
```

Comment:
I am not sure that this fits under "Mild unprofessionalism" and should also mention relationships and depression. In our rules, we have it under the "This community only supports the content in our curriculum" rule. However, if we feel strongly that it needs to stay here, then I wonder if we should also move it in our rules.

I also think it should be moved under the "A second infraction" or 0 zaps for first-time offenders.

Reply:
I think depression falls under the last point. For me, all of these topics would probably cause discomfort in the workplace.

Reply:
I agree @twalton83 , I had a talk about this with my manager at work and she did say that people airing our their personal issues in a semi-public place where people have to do business is unprofessional. Relationships should for sure be added.